_layouts/user_story.html

```diff
@@ -20,15 +20,23 @@
 <section class="panel">
   <div class="post-content">
     {% if page.image %}
-    <img
-      class="post-content__featured-image"
-      srcset="{{ page.image | append: '?nf_resize=fit&w=480 480w' }},
-                {{ page.image | append: '?nf_resize=fit&w=720 720w' }},
-                {{ page.image | append: '?nf_resize=fit&w=1200 1200w' }}"
-      src="{{ page.image | append: '?nf_resize=fit&w=1200' }}"
-    />
+      <picture>
```

Comment:
Could you explain to me
- how picture, source and img work together?
- how nf_resize and w work in the query for srcset?
Seems cool :)

Reply:
- from the mdn docs the <picture> element:
> The HTML `<picture>` element contains zero or more `<source>` elements and one `<img>` element to offer alternative versions of an image for different display/device scenarios. 
>
> The browser will consider each child `<source>` element and choose the best match among them. If no matches are found—or the browser doesn't support the `<picture>` element—the URL of the `<img>` element's src attribute is selected. The selected image is then presented in the space occupied by the `<img>` element

more here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

- with Netlify Large Media you can request a transformed image. `nf_resize=fit&w=1200` in the query means give me this image resized to fit 1200px width. see more here: https://docs.netlify.com/large-media/transform-images/#request-transformations

Reply:
Cool, thanks!